tests/cases/fourslash/quickInfoJsDocTags14.ts

```diff
@@ -0,0 +1,11 @@
+/// <reference path="fourslash.ts" />
+
+/////**
+//// * @param a - a param
+/////* @param newA - new a param
+//// */
+////function foo({ a: newA }: { a: string }) {
+////    newA/*1*/
```

Comment:
I would expect the typical usage would be just `@param a`, since that's the name that callers must provide. 
However, the type of `newA` should still be string, even in JS (so I think a JS test is needed too, unless there already is one)